angr/engines/concrete.py

```diff
@@ -0,0 +1,158 @@
+import logging
+import threading
+
+from angr.errors import AngrError
+from .engine import SimEngine
+from ..errors import SimConcreteMemoryError, SimConcreteRegisterError
+
+l = logging.getLogger("angr.engines.concrete")
+# l.setLevel(logging.DEBUG)
+
+try:
+    from angr_targets.concrete import ConcreteTarget
+except ImportError:
+    l.error("Can't find angr_targets project!")
```

Comment:
We shouldn't print an error by default if an optional dependency is not installed - we should instead raise an error when someone tries to actually use it. Remember the bad old days when there was a warning when you didn't have the special version of capstone installed every time you import angr? Don't do that again.